modules/ROOT/pages/admin-console-VM.adoc

```diff
@@ -0,0 +1,267 @@
+= Install Pulsar Admin Console on Server/VM
+
+The Pulsar Admin Console has 2 main components: 
+
+* A Node.js application that runs the server and serves the dashboard client code to browsers. 
+* An NGINX server that routes network traffic to different locations within the Pulsar Cluster. 
+
+Technically, the two processes do not need to be collocated, but we recommend running them on the same host/VM.
+
+== Install Pulsar Admin Console
+
+. Download and install the Node version 11.15.0 tarball to the VM. 
+
```

Comment:
Can we include a link to the node.js download page so that users can find the correct version to download for their OS. Here is the link: https://nodejs.org/en/download/